frontend/packages/rhoas-plugin/src/catalog/providers/useRhoasCatalog.tsx

```diff
@@ -49,72 +42,88 @@ const useRhoasCatalog: CatalogExtensionHook<CatalogItem[]> = ({
+          <FlexItem className="catalog-tile-pf-description">
+            {t(
+              'rhoas-plugin~Red Hat OpenShift Application Services include Red Hat OpenShift API Management and Red Hat OpenShift Streams for Apache Kafka',
+            )}
           </FlexItem>
           <FlexItem>{token}</FlexItem>
         </Flex>
       );
     };
 
-    const serviceKafkaCardDescription = (
-      <TextContent>
-        <Text component={TextVariants.p}>{t('rhoas-plugin~KafkaCardDescription')}</Text>
-      </TextContent>
-    );
-
-    const serviceKafkaCardDetailsDescription = [
-      {
-        value: serviceKafkaCardDescription,
-      },
-    ];
-
     const cloudServicesCardDetailsDescription = [
       {
-        label: t('rhoas-plugin~Access Red Hat Cloud Services with API Token'),
+        label: t('rhoas-plugin~Unlock with API token'),
         value: <ServiceToken namespace={namespace} />,
       },
       {
         value: <Divider component="li" />,
       },
       {
         label: 'Description',
-        value: t('rhoas-plugin~ManagedServices-card-description'),
+        value: (
+          <Text component={TextVariants.p}>
+            {t(
+              'rhoas-plugin~Red Hat OpenShift Application Services include:\n- Red Hat OpenShift API Management\n- Red Hat OpenShift Streams for Apache Kafka',
+            )}
+          </Text>
+        ),
+      },
+      {
+        value: (
+          <Text component={TextVariants.p}>
+            {t('rhoas-plugin~Cloud Services Card Description')}
+          </Text>
+        ),
       },
     ];
 
     if (isServiceAccountValid) {
-      const serviceKafkaCard: CatalogItem[] = [
-        {
-          name: t('rhoas-plugin~KafkaStreamsService'),
-          type: CATALOG_TYPE,
-          uid: 'streams-1615213269575',
-          description: t('rhoas-plugin~KafkaStreamsServiceDescription'),
-          provider: 'Red Hat',
-          tags: ['kafka'],
-          icon: {
-            url: kafkaIcon,
-          },
-          cta: {
-            label: t('rhoas-plugin~Connect'),
-            href: `/rhoas/ns/${namespace}/kafka`,
-          },
-          details: {
-            descriptions: serviceKafkaCardDetailsDescription,
-          },
+      const rhoasCard: CatalogItem[] = RHOASServices.map(
+        ({
+          serviceName,
+          name,
+          type,
+          uid,
+          description,
+          provider,
+          tags,
+          icon,
+          ctaLabel,
+          details,
+        }) => {
+          return {
+            name,
+            type,
+            uid,
+            description,
+            provider,
+            tags,
+            icon: {
+              url: icon,
+            },
+            cta: {
+              label: ctaLabel,
+              href: `/rhoas/ns/${namespace}/${serviceName}`,
+            },
+            details: {
+              descriptions: [{ value: <Text component={TextVariants.p}>{details}</Text> }],
```

Comment:
Why using TextVatiants.P? maybe that is the reason why markdown doesn't work? Not sure myself